packages/terra-tabs/tests/jest/Tabs.test.jsx

```diff
@@ -7,62 +7,79 @@ describe('Tabs', () => {
   it('should render a default component', () => {
     const defaultRender = <Tabs><Tabs.Pane label="Default" key="default" /></Tabs>;
     const wrapper = shallowWithIntl(defaultRender);
+    wrapper.update();
     expect(wrapper).toMatchSnapshot();
   });
 
   describe('Variants', () => {
     it('should render modular-left-aligned tabs', () => {
       const defaultRender = <Tabs variant="modular-left-aligned"><Tabs.Pane label="Default" key="default" /></Tabs>;
       const wrapper = shallowWithIntl(defaultRender);
+      wrapper.setState({ showCollapsedTabs: true });
```

Comment:
I believe that these "Variants" tests could be removed - there is no `variant` prop.

Reply:
@nramamurth should we log an issue to clean it up?

Reply:
@emilyrohrbough I remember that there was a discussion to remove modular tabs implementation on one of the PRs and Ryan had mentioned that we could do that as a separate issue. I am not sure if we have an issue logged to do that though. I shall go ahead and create one and document the this detail as well if there ain’t one.